app/components/submission/ManuscriptMetadata/ManuscriptMetadata.js

```diff
@@ -48,6 +54,16 @@ const ManuscriptMetadata = ({ handleSubmit, values, setFieldValue }) => (
       ]}
     />
 
+    <ValidatedField
+      component={SubjectAreaDropdown}
+      label="Subject areas"
+      name="subjectAreas"
+      onBlur={e => setFieldTouched('subjectAreas', true)}
```

Comment:
You mean inside `onChange` as well?

Nah, when a field's value is changed (i.e. using `setFieldValue` in the onChange handler), formik changes `touched` to `true` under the hood :+1: 

But on blur, we need to "set the state of touched imperatively", [as it says in formik's docs](https://github.com/jaredpalmer/formik#setfieldtouched-field-string-istouched-boolean-shouldvalidate-boolean--void), so that it will get picked up by the validation on submit.

Tbh I don't feel like this is paticularly clear from the documentation & kinda requires you to understand the internals of formik. I was thinking of raising an issue on it, on their repo :thinking: , but I'm unsure how they should provide more clarity.

Fortunately, [this example with `react-select](https://codesandbox.io/s/jRzE53pqR) is linked to in Formik's documentation, which was super helpful :grin: It essentially showed me what I needed to write, but I didn't really understand _why_ until I watched [this youtube video](https://www.youtube.com/watch?v=-tDy7ds0dag&t=957s) (where the creator of the repo talks things through), or whether it was definitely the thing I ought to be doing.

Hopefully those links will help you if you're ever grappling with formik yourself. It was one of many things that stumped me for a while, but results in only a short line of code to show for it. Such is the way sometimes :laughing:

Hope that helps?

Reply:
Ooo thanks very much - super helpful!